srcpkgs/gns3-server/template

```diff
@@ -13,16 +13,16 @@ maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://gns3.com"
 distfiles="https://github.com/GNS3/gns3-server/archive/v${version}.tar.gz"
-checksum=@9f3d7f4caf6cd420049a62ae9d7579f29b0e3805f1a50913ae9111ab6752bdb0
+checksum=0ed75a0d4438ef5cc07cccbccb79a60e6982f615981a58a2dde5082773203575
 make_check=no
 
 post_patch() {
 	# relax requirements
-	vsed -e 's|aiofiles.*|aiofiles>=0.4.0|' \
+	vsed -e 's|Jinja2.*|Jinja2<3.0.0|' \
```

Comment:
Upstream gns3-server was updated to use Jinja2 version 3.0 but some of the void packages do not work with the new version yet (e.g. https://github.com/saltstack/salt/issues/60188).  The requirements file is updated to continue allowing use of Jinja2 2.x until the 3.x version is usable by other packages.